services/state/schedule/schedule.proto

```diff
@@ -155,10 +153,40 @@ message ScheduleList {
  * One example is all schedules that expire at a particular time.
  */
 message ScheduleIdList {
+  /**
+   * A list of schedule identifiers, in no particular order.
+   * <p>
+   * While the order is not _specified_, it MUST be deterministic.
+   */
+  repeated ScheduleID schedule_ids = 1;
+}
+
+/**
+ * The value of a map summarizing the counts of scheduled and processed transactions
+ * within a particular consensus second.
```

Comment:
Might need to be a different PR, but the specification text for this message and the next needs to be updated to specification standards rather than simple description.